docs/source/commands/persisted-queries.mdx

```diff
@@ -54,3 +54,10 @@ rover persisted-queries publish \
 2. Updates any _other_ variants that the PQL is applied to so that routers associated with those variants can fetch their updated PQL.
 
 > As with [generating manifests](/graphos/operations/persisted-queries#31-generate-persisted-queries-manifests), it's best to execute this command in your CI/CD pipeline to publish new operations as part of your app release process. This command requires your API key to have the [**Graph Admin** or **Persisted Query Publisher** role](/graphos/org/members#graph-api-key-roles); the latter is a role specifically designed to work with this command without exposing unnecessary information about your graph to users of the API key.
+
+#### Relay support
+
+The `rover persisted-queries` command supports publishing [persisted queries generated by the Relay compiler](https://relay.dev/docs/guides/persisted-queries/).
```

Comment:
I think using this anchor from the beginning is best since it points readers straight to the relevant section. They can always scroll up if they want more info about Relay's PQ functionality.

```suggestion
The `rover persisted-queries` command supports publishing [persisted queries generated by the Relay compiler](https://relay.dev/docs/guides/persisted-queries/#local-persisted-queries).
```